ql/src/java/org/apache/hadoop/hive/ql/exec/vector/VectorSerializeRow.java

```diff
@@ -383,6 +414,13 @@ private void serializeMapWrite(
     serializeWrite.finishMap();
   }
 
+  class VectorSerializeListWriter extends VectorSerializeWriter {
+    @Override
+    void serialize(Object colInfo, Field field, int adjustedBatchIndex) throws IOException {
+      serializeListWrite((ListColumnVector)colInfo, field, adjustedBatchIndex);
+    }
+  }
+
   private void serializeListWrite(
```

Comment:
Nit: serializeListWrite part of VectorSerializeListWriter?

Reply:
done